src/lib/descriptor_test.rs

```diff
@@ -604,11 +604,28 @@ fn load_external_descriptor_set_env() {
     envmnt::set("CARGO_MAKE_MAKEFILE_PATH", "EMPTY");
     assert_eq!(envmnt::get_or_panic("CARGO_MAKE_MAKEFILE_PATH"), "EMPTY");
 
-    load_external_descriptor(".", "./examples/alias.toml", true, true);
+    load_external_descriptor(".", "./examples/alias.toml", true, true).unwrap();
 
     assert!(envmnt::get_or_panic("CARGO_MAKE_MAKEFILE_PATH").ends_with("alias.toml"));
 }
 
+#[test]
+fn load_external_descriptor_min_version_broken_makefile_nopanic() {
+    // Ensure that we can properly get the min_version of a descriptor that
+    // doesn't match our internal data-structures.
+    assert_eq!(
+        load_external_descriptor(".", "src/lib/test/makefiles/broken_makefile_minversion.toml", false, false).err(),
+        Some("999.999.999".into()));
+}
+
+#[test]
+#[should_panic]
+fn load_external_descriptor_broken_makefile_panic() {
+    // Ensure the descriptor we used in the test above is properly broken when
+    // it doesn't have a min_version
+    load_external_descriptor(".", "src/lib/test/makefiles/broken_makefile.toml", false, false).unwrap();
```

Comment:
No. This means that if min_version is not present and we load a makefile that does not parse with the current version of cargo-make, a panic will happen. This test is meant to ensure that we successfully parse min_version **before** parsing anything that is not accepted by the current parser. It's an extension of the nomin test.

Reply:
To comment a bit further: the makefile is broken because `command = false` is not deserializable with the internal data structures, *not* because min_version is not present.